src/olympia/addons/tasks.py

```diff
@@ -217,6 +217,7 @@ def recreate_theme_previews(addon_ids, **kw):
                 all_full_sizes_present = not set(renders.keys()) - existing_full_sizes
                 if all_full_sizes_present:
                     # i.e. we have all renders
+                    log.info('Resizing thumbnais for theme: %s' % version.addon_id)
```

Comment:
```suggestion
                    log.info('Resizing thumbnails for theme: %s' % version.addon_id)
```

Reply:
```suggestion
                    log.info('Resizing thumbnais for theme: %s', version.addon_id)
```

seems a bit better because Sentry will aggregate the logs.

Reply:
info level logging doesn't show up in Sentry - only error/exception